### ISSUE TITLE
Adding number of parameters / complexity to the table

### ISSUE THREAD

**Issue by ialhashim** [Cadene/pretrained-models.pytorch#102]
Great resource, thanks for your efforts. 

Is it possible to also include the number of parameters along with the ImageNet accuracy?

Also, a side note, have you considred ShuffleNetV2? here is a great implementation with two pretrained models https://github.com/ericsun99/Shufflenet-v2-Pytorch
